tox.ini

```diff
@@ -15,7 +15,7 @@ passenv =
     LOG_FORMATTER
     LOG_THREAD_ID
     LOG_CHANNEL_WIDTH
-commands = pytest --cov=caikit --cov-report=html {posargs:tests}
+commands = pytest --cov=caikit --cov-report=html:coverage-{env_name} --cov-report=xml:coverage-{env_name}.xml {posargs:tests}
```

Comment:
Nice!

Can we get a .gitignore entry for the coverage xml files?

Reply:
Done